content/docs/developing-operators.md

```diff
@@ -126,10 +138,12 @@ Another part of `operator.yaml` is the tasks section. Tasks are the smallest pie
 
 ```yaml
 tasks:
-  deploy-task:
-    resources:
-      - config.yaml
-      - pod.yaml
+  - name: app
+    kind: Apply
+    spec:
+      resources:
+        - pod.yaml
+        - config.yaml
 ```
 
```

Comment:
has in the doc a typo issue `kudoctl kudo get instances` instead of `kubectl kudo get instances`